Packs/RubrikPolaris/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,70 @@
+
+#### Integrations
+##### Rubrik Radar
+- Modified the graphql query during incident ingestion to filter on the 'lastUpdated' time for new incidents in accordance to backend functionality changes to Radar.  This also fixed a bug where new incidents were not being detected by XSOAR.
+- Added two new options in the Rubrik Polaris integration settings that allow the user to map the Radar Critical and Warning event severity levels to various XSOAR severity levels upon incident fetch.  The default severity mappings are currently:  Radar Critical -> XSOAR HIGH and Radar Warning -> XSOAR LOW.
+- Added the 'rubrik-cdm-cluster-connection-state' and 'rubrik-cdm-cluster-location' commands.  These commands have been added in as tasks to the Rubrik Post Intrusion Ransomware Investigation playbook.
+- Added the Rubrik CDM Cluster Location and Rubrik CDM Cluster Name custom fields.
+- Added functionality to pull in the Radar File Change attributes from Radar events during incident ingestion.
+- Various layout changes were made to the Rubrik Radar Incident Summary page.
+- Renamed the Rubrik Polaris CDM Cluster ID field to Rubrik CDM Cluster ID and modified the associated mapper.
```

Comment:
From what I gather you have deprecated the old fields and added new ones - please mention that in the `#### Incident Fields` section below - I'll leave a comment there as well.
please remove this line though